ucsdscheduleme/ucsdscheduleme.csproj

```diff
@@ -13,6 +13,6 @@
   </ItemGroup>
 
   <ItemGroup>
-    <Compile Remove="Controllers\ScheduleController.cs" />
```

Comment:
Are you sure that this should be removed here? This should cause some problems. Can you do a clean build in VS and see if this causes a complaint?

Reply:
Did a clean build and VS did not complain.